lib/local-links-manager/check_links/homepage_status_updater.rb

```diff
@@ -21,7 +21,7 @@ def update
     private
 
       def links
-        table.distinct.pluck(column)
+        table.distinct.where.not(column => '').pluck(column)
```

Comment:
Only a thought, the commit message mentions that this method will only retrieve links that are not set to nil and that are not set to an empty string. The added code sort of looks like it only checks for empty strings (rather than something like `.where.not(column => [nil, ''])`. However the added test seems to demonstrate that it also works for homepages that are set to nil, so it's probably fine? I'm assuming this is due to the fact that `nil` in Postgres is stored as an empty string... I might have to read up on this :)